### ISSUE TITLE
Add label to requestable select elements

### ISSUE THREAD

**Issue by axamei** [pulibrary/requests#348]
Violation: Form elements must have labels.

```
<select name="requestable[][type]" class="request-options"> <option value="">Select a Request Method</option> <option value="bd">Borrow Direct (4 business days or less)</option> <option value="ill">Interlibrary Loan (1-2 weeks)</option> </select>
```

Add `<label>` or aria-label or aria-labelledby
